dronepid_node:
  ros_paramters:
    kp: 1
    ki: 0.0
    kd: 0.1

    resX: 640
    resY: 480
    